workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -194,73 +127,28 @@ internal static IEnumerable<BuildTarget> GetUnityBuildTargets(SpatialBuildPlatfo
                 var result = BuildPipeline.BuildPlayer(buildPlayerOptions);
                 if (result.summary.result != BuildResult.Succeeded)
                 {
-                    throw new BuildFailedException($"Build failed for {workerPlatform}");
+                    throw new BuildFailedException($"Build failed for {workerType}");
                 }
 
-                var zipPath = Path.GetFullPath(Path.Combine(PlayerBuildDirectory, workerBuildData.PackageName));
-
-                var basePath = PathUtil.Combine(BuildPaths.BuildScratchDirectory, workerBuildData.PackageName);
-
-                Zip(zipPath, basePath,
-                    targetEnvironment == BuildEnvironment.Local
-                        ? PlayerCompression.Disabled
-                        : PlayerCompression.Enabled);
+                var zipPath = Path.Combine(PlayerBuildDirectory, workerBuildData.PackageName);
+                var basePath = PathUtils.Combine(PathUtils.BuildScratchDirectory, workerBuildData.PackageName);
+                Zip(zipPath, basePath, targetEnvironment == BuildEnvironment.Local);
             }
             finally
             {
-                PlayerSettings.SetScriptingDefineSymbolsForGroup(BuildTargetGroup.Standalone, symbols);
+                PlayerSettings.SetScriptingDefineSymbolsForGroup(buildTargetGroup, symbols);
             }
         }
 
-        private static void Zip(string zipAbsolutePath, string basePath, PlayerCompression useCompression)
+        private static void Zip(string zipAbsolutePath, string basePath, bool useCompression)
         {
-            var zipFileFullPath = Path.GetFullPath(zipAbsolutePath);
-
             using (new ShowProgressBarScope($"Package {basePath}"))
             {
-                RedirectedProcess.Run(Common.SpatialBinary, "file", "zip", $"--output=\"{zipFileFullPath}\"",
+                RedirectedProcess.Run(Common.SpatialBinary, "file", "zip", 
```

Comment:
As an aside we will need to replace this for FPL

Reply:
yeah, I know. but we will have to do a full pass for code gen as well :(